invenio_db/cli.py

```diff
@@ -8,23 +8,13 @@
 
 """Click command-line interface for database management."""
 
-import sys
-
 import click
-from click import _termui_impl
-from flask import current_app
 from flask.cli import with_appcontext
 from sqlalchemy_utils.functions import create_database, database_exists, drop_database
-from werkzeug.local import LocalProxy
 
+from .proxies import current_db
```

Comment:
In general, what we put in the `proxies`, is our Invenio extensions. Shall we keep this proxy inside here, private, so that there is no confusion of what is `current_db`?
When I see `current_db`, the first thought that I have is that it is `invenio_db`, and not a reference to `sqlalchemy`.

Reply:
hm, good point. Maybe that was also the reason it was here in the first place. I created the proxies.py file. OK i will change that back.